src/WebJobs.Script.Grpc/Server/IHttpProxyService.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Yarp.ReverseProxy.Forwarder;
+
+namespace Microsoft.Azure.WebJobs.Script.Grpc
+{
+    public interface IHttpProxyService
+    {
+        public ValueTask<ForwarderError> ForwardAsync(ScriptInvocationContext context, Uri httpUri);
```

Comment:
small nit: don't need to include `public` on interfaces, they are public by default.
```suggestion
        ValueTask<ForwarderError> ForwardAsync(ScriptInvocationContext context, Uri httpUri);
```